docs/parameters.yaml

```diff
@@ -128,6 +128,7 @@ description: >-
   will result in the IP address "123.234.123.234" being mapped to Madison, WI, and IP addresses in the range ABCD::0000-FFFF will be mapped
   to a field in Kansas.
 type: object
+objectStructure: GeoIPOverride
```

Comment:
I would suggest for the object parameters, we use parameter name for their object type instead of having a separate field that's mostly the same name as the parameter name, unless it's needed for something else.

Reply:
@haoming29 So are you saying in this case it would be

type: "GeoIPOverride"

Reply:
Nah, I meant to remove `objectStructure` field from `parameters.yaml` and in the python script, use `GeoIPOverride` as the item for `VERIFIED_OBJECT_STRUCTURES`. Similarly, we can do `Registry.Institutions`, `Issuer.OIDCAuthenticationRequirements` as the other items. Does that make sense?

Reply:
Great idea, I changed it to read the object name from the end of the period delimited string so we don't have to duplicate things. This also allows another area to use the  Institutions object without additional work.